src/util/analytics-client.ts

```diff
@@ -33,7 +32,7 @@ export function sendAnalytics(eventType: string, attributes: Record<string, unkn
 		tenantId: "NONE"
 	};
 
-	attributes.appKey = appKey;
+	attributes!.appKey = appKey;
```

Comment:
Some deja vu here, is this `attributes` optional or not? Why the `!` here?

Reply:
yeah, agreed.  Might be easier to have the parameter be `attributes: Record<string, unknown> = {}` so that it's never undefined